app/views/produce_items/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Items</h1>
+
+<table>
+    <% @produce_items.find_each do |produce|  %>
+      <tr>
+        <td><%= I18n.t("produce_items.names.#{produce.name}") %></td>
+        <td><%= I18n.t("produce_items.categories.#{produce.category}") %></td>
+        <td><%= link_to 'ver', produce %></td>
```

Comment:
Should we move hardcoded strings to our `pt-BR.yml` file?

Reply:
you mean like 'show', 'back' etc right? makes sense